src/components/local-block.tsx

```diff
@@ -103,7 +103,8 @@ export const LocalBlock = (props: LocalBlockProps) => {
       metadata={metadata}
       onUpdateMetadata={onUpdateMetadata}
       onNavigateToPath={onNavigateToPath}
-      onRequestUpdateContent={setContent}
+      onUpdateContent={setContent}
+      onRequestUpdateContent={setContent} // for backward compatibility
```

Comment:
are we safe to remove this, since we're in our own examples repo? or are we still waiting on another update?

Reply:
yeah I guess you are right, with all the example blocks updated it should be fine.